### DISCUSSION TITLE
does wz_mini_hacks  supports any kind of wifi usb stiks?

### DISCUSSION THREAD

**Discussion by johnvprcam24573455623** [gtxaspec/wz_mini_hacks#585] [General]
I have a camera far away from my AP,  I am wondering if I can connect a wifi usb stik with a usb extension to extend the wifi range

**Comment by Bob-AOL** [comment]
How are you powering the camera?

**Comment by bjs-pdx** [comment]
" I am wondering if I can connect a wifi usb stik with a usb extension"

Is there a reason you cannot or do not want to use ethernet and POE?
